docs/reference/how-to/use-elasticsearch-for-time-series-data.asciidoc

```diff
@@ -60,17 +58,13 @@ A <<data-streams,data stream>> stores your data across multiple backing
 indices. {ilm-init} uses an <<ilm-index-lifecycle,index lifecycle policy>> to
 automatically move these indices through your data tiers.
 
-A lifecycle policy contains phases that correspond to data tiers. You can
-specify automated actions to take on indices in each phase. A policy can also
-include a delete phase to safely delete indices you no longer need.
```

Comment:
I removed this as it was unneeded explainer text. We link to the lifecycle concept docs above. The list below points out the relationship between tiers and phases and mentions the delete phase.